.doc_gen/readmes/README.md

```diff
@@ -20,9 +20,10 @@ and install packages by running the following commands in the
 `.doc_gen/readmes` folder:
 
 ```
-python -m venv venv
-venv/Scripts/activate
-python -m pip install -r requirements.txt
+python -m venv .venv
+.venv\Scripts\activate # Windows
+source .venv/bin/activate # *nix
+pip install -r requirements.txt
```

Comment:
Current recommendation is to [run pip as a module](https://realpython.com/what-is-pip/#running-pip-as-a-module), so:
```
python -m pip install -r requirements.txt
```